eZ/Publish/Core/settings/fieldtype_services.yml

```diff
@@ -106,3 +106,9 @@ services:
             $locationService: '@ezpublish.api.service.location'
             $contentTypeService: '@ezpublish.api.service.content_type'
             $mappings: '$fieldtypes.ezimageasset.mappings$'
+
+    # Relation and RelationList
+    ezpublish.fieldType.validator.destination_content:
+        class: eZ\Publish\Core\FieldType\Validator\DestinationContentValidator
```

Comment:
Do not use named services please. It's redundant. Use FQCN and follow the rule mentioned in the previous diff.